weave/artifact_wandb.py

```diff
@@ -1068,6 +1092,113 @@ def to_ref(self) -> WandbArtifactRef:
         return WandbArtifactRef.from_uri(self)
 
 
+@dataclasses.dataclass
+class WeaveWBArtifactByIDURI(uris.WeaveURI):
```

Comment:
Do you mind adding a comment to what this URI is and why it is special?

Reply:
✅